en/deploy/README.md

```diff
@@ -130,159 +130,106 @@ Your code is now on GitHub. Go and check it out!  You'll find it's in fine compa
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 ```
-$ tree my-first-blog
-my-first-blog/
-├── blog
-│   ├── __init__.py
-│   ├── admin.py
-│   ├── migrations
-│   │   ├── 0001_initial.py
-│   │   └── __init__.py
-│   ├── models.py
-│   ├── tests.py
-│   └── views.py
-├── manage.py
-└── mysite
-    ├── __init__.py
-    ├── settings.py
-    ├── urls.py
-    └── wsgi.py
+$ pip3.6 install --user pythonanywhere
 ```
 
+That should print out some things like `Collecting pythonanywhere`, and eventually end with a line saying `Successfully installed (...) pythonanywhere- (...)`.
 
-### Creating a virtualenv on PythonAnywhere
-
-Just like you did on your own computer, you can create a virtualenv on PythonAnywhere. In the Bash console, type:
+Now we run the helper to automatically configure our app from GitHub. Type the following into the console on PythonAnywhere (don't forget to use your GitHub username in place of `<your-github-username>`):
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 ```
-$ cd my-first-blog
-
-$ virtualenv --python=python3.6 myvenv
-Running virtualenv with interpreter /usr/bin/python3.6
-[...]
-Installing setuptools, pip...done.
-
-$ source myvenv/bin/activate
-
-(myvenv) $  pip install django~=1.11.0
-Collecting django
-[...]
-Successfully installed django-1.11.3
+$ pa_autoconfigure_django.py https://github.com/<your-github-username>/my-first-blog.git
 ```
 
+As you watch that running, you'll be able to see what it's doing:
 
-> **Note** The `pip install` step can take a couple of minutes.  Patience, patience!  But if it takes more than five minutes, something is wrong.  Ask your coach.
-
-<!--TODO: think about using requirements.txt instead of pip install.-->
+- Downloading your code from GitHub
+- Creating a virtualenv on PythonAnywhere, just like the one on your own PC
+- Updating your settings file with some deployment settings
+- Setting up a database on PythonAnywhere using the `manage.py migrate` command
+- Setting up your static files (we'll learn about these later)
+- And configuring PythonAnywhere to serve your web app via its API
 
-### Creating the database on PythonAnywhere
+On PythonAnywhere all those steps are automated, but they're the same steps you
+would have to go through with any other server provider.  The main thing to notice
+right now is that your database on PythonAnywhere is actually totally separate from
+your database on your own PC—that means it can have different posts and admin accounts.
 
-Here's another thing that's different between your own computer and the server: it uses a different database. So the user accounts and posts can be different on the server and on your computer.
-
-Just as we did on your own computer, we repeat the step to initialize the database on the server, with `migrate` and `createsuperuser`:
+As a result, just as we did on your own computer, we repeat the step to
```

Comment:
While it can be concluded that we have to repeat that step, doing so is not a "result" of what was mentioned above. <del>As a result,</del>Thus, maybe use
> Thus, [...]

instead of 
> As a result, [...]

Reply:
can i disagree with this?  to me the meaning of "as a result" is almost exactly the same as "thus". we have to run createsuperuser again *because* the database is separate / as a result of the fact that the database is seperate / the database is seperate, thus we have to run createsuperuser again.  they all mean the same thing to me.

Reply:
Ah, then maybe include the "have to":
> As a result, just as we did on your own computer, we **have to** repeat the step to [...]

Reply:
never mind, I now see you already [changed it to "need to"](https://github.com/DjangoGirls/tutorial/commit/44b7f9d#diff-733b3a4f3942013e4c0f308cd2a471acR185), which is just as fine to make to logic work out